magic-nix-cache/src/pbh.rs

```diff
@@ -88,6 +88,8 @@ pub async fn subscribe_uds_post_build_hook(
                     continue;
                 };
 
+                tracing::trace!(?event, "Built path event received");
```

Comment:
we trace debug anyway like 20 lines down.

Reply:
I added it here in case it was being eliminated by a `continue` below

Reply:
fair - (fwiw we would've seen an error line if that was the case, and I didn't see it)